airflow/jobs/scheduler_job.py

```diff
@@ -863,10 +862,9 @@ def _find_executable_task_instances(self, simple_dag_bag, states, session=None):
         for task_instance in task_instances_to_examine:
             pool_to_task_instances[task_instance.pool].append(task_instance)
 
-        states_to_count_as_running = [State.RUNNING, State.QUEUED]
         # dag_id to # of running tasks and (dag_id, task_id) to # of running tasks.
         dag_concurrency_map, task_concurrency_map = self.__get_concurrency_maps(
-            states=states_to_count_as_running, session=session)
+            states=STATES_TO_COUNT_AS_RUNNING, session=session)
 
         # Go through each pool, and queue up a task for execution if there are
         # any open slots in the pool.
```

Comment:
Isn't the code below here (handling Pools) now handled by a Dep so this should be removed?